ivy/functional/frontends/numpy/ndarray/ndarray.py

```diff
@@ -471,3 +471,13 @@ def __setitem__(self, key, value):
 
     def __mod__(self, value, /):
         return np_frontend.mod(self._ivy_array, value, out=self)
+    
+    def ptp(self, *, axis=None, dtype=None, out=None, keepdims=False, where=True):
+        return np_frontend.ptp(
```

Comment:
There is no method `ptp` present in the `np_frontend` namespace. Did you try running any tests locally to confirm that they pass? The [CI tests](https://github.com/unifyai/ivy/actions/runs/4632978419/jobs/8197683085?pr=13696#step:3:123) fail essentially because of this. If there is no function `ptp` in the `np_frontend` or `ivy` namespace, then you will have to implement the logic of your function as a composition of other `ivy` functions such as `ivy.max`, `ivy.min` etc.